cfgov/search/models/models.py

```diff
@@ -0,0 +1,4 @@
+from django.db import models
+
+class Synonym(models.Model):
+    synonym = models.CharField(max_length=500)
```

Comment:
We may want to consult with content editors, but a `language` field might be handy for editors who want to see synonyms for a particular language.